universal/ubi8/Dockerfile

```diff
@@ -81,6 +81,30 @@ RUN dnf -y module enable python39:3.9 && \
     dnf -y install python39 python39-devel python39-setuptools python39-pip nss_wrapper && \
     dnf -y clean all --enablerepo='*'
 
+# PHP
+ENV PHP_VERSION=7.4
```

Comment:
maybe we should use 8.0 as 7.4 is no longer active in a couple of days
https://www.php.net/supported-versions.php

Reply:
it seems anyway 8.0 is not in ubi8 so...

Reply:
@benoitf why I've decided to install 7.4, because it is the latest version which is used in RedHat images https://catalog.redhat.com/software/containers/search?q=php&p=1